src/xcode/ENA/ENA/Source/Views/CardView.swift

```diff
@@ -8,9 +8,40 @@ class CardView: UIView {
 
 	// MARK: - Overrides
 
-	override func awakeFromNib() {
-		super.awakeFromNib()
+	override init(frame: CGRect) {
+		super.init(frame: frame)
+		setup()
+	}
+
+	required init?(coder: NSCoder) {
+		super.init(coder: coder)
+		setup()
+	}
```

Comment:
Should go to a `// MARK: - Init` section

Reply:
👍